projects/sandbox/generate_background/generate_background.py

```diff
@@ -63,9 +63,11 @@ def main(
         stop,
     )
 
+    print(segments)
```

Comment:
Were these handful of print statements just left over from debugging? Wanted to flag them just in case